### PR TITLE
added build command to travis script

### PR DESCRIPTION
**Pull request by davodey**
I think this will fix the problem of the build folder not being generated and deployed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,7 @@ node_js:
 script:
 - export REPO_NAME=glp-elements-sdk
 - export SUITE_NAME=glp_elements_sdk
-after_success:
-- npm run deploy-ci
+- npm run build
```
Comment (mhomolak) [inline .travis.yml]: Might be a good idea to just add back the ci script.  On top of the build it gives you a nice little sanity check that runs the unit tests and will fail a release if any aren't passing.

Comment (davodey) [reply .travis.yml]: Yeah, that sounds good.

File: .travis.yml [hunk 2]
```diff
@@ -3,9 +3,10 @@ sudo: false
 node_js:
 - "stable"
 script:
-- export REPO_NAME=glp-elements-sdk
```
Comment (davodey) [inline .travis.yml]: @mhomolak do we need these variables?

Comment (umahaea) [reply .travis.yml]: since selenium ci tests run is turned off, we don't need them.

Comment (mhomolak) [reply .travis.yml]: also the after_success: script isn't linked up to anything currently so that could be removed.

**Pull request merged**